web-components/src/components/form/RecipientsForm.tsx

```diff
@@ -38,11 +44,17 @@ export interface FormValues {
   recipients: Recipient[];
 }
 
-const validationSchema = Yup.object().shape({
+export const validationSchemaFields = {
   recipients: Yup.array()
     .of(
       Yup.object().shape({
-        recipient: Yup.string().required(requiredMessage), // TODO: improve validation: .min(20, 'too short').required
+        recipient: Yup.string()
+          .required(requiredMessage)
+          .test('is-regen-address', invalidRegenAddress, value =>
+            typeof value === 'string'
+              ? validateAddress(value, 'regen') // hardcoded prefix `regen`
```

Comment:
best to pass this as a prop and use `chainInfo.bech32Config.bech32PrefixAccAddr` from web-registry

Reply:
ok, it was my intention at first but then I didn't see it completely clear and I thought I was getting complicated. I already have a proposal, wrapping the validation schema object with a function to complete the schema